migrations/0034-add-userSocialMediaUrl-table.ts

```diff
@@ -0,0 +1,48 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+export class AddUserSocialMediaUrlsTable1663760514239 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(new Table({
+      name: 'UserSocialMediaUrls',
+      columns: [
+        {
+          name: 'uuid',
+          type: 'uuid',
+          isPrimary: true,
+          isGenerated: true,
+          generationStrategy: 'uuid',
+        },
+        {
+          name: 'user',
+          type: 'uuid',
+        },
+        {
+          name: 'socialMediaType',
+          type: 'varchar(255)',
+        },
+        {
+          name: 'url',
+          type: 'varchar(255)',
+        },
+      ],
+      indices: [
+        {
+          name: 'social_media_url_by_user_index',
+          columnNames: ['user'],
+        },
+      ],
+      foreignKeys: [
+        {
+          columnNames: ['user'],
+          referencedTableName: 'Users',
+          referencedColumnNames: ['uuid'],
+          onDelete: 'CASCADE',
+        },
+      ],
+    }));
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.dropTable('UserSocialMediaUrls');
+  }
```

Comment:
nit: generally in migrations we have a `TABLE_NAME` constant that holds the table name, and we use that in the createTable and dropTable functions to avoid repetition